src/main/resources/content/tf/AWS-Access-logging-not-enabled-on-S3-buckets.json

```diff
@@ -0,0 +1,8 @@
+{
+  "severity": "medium",
+  "rule": "$.resource[*].aws_s3_bubcket exists and ($.resource[*].aws_s3_bubcket[*].*[*].logging anyNull)",
```

Comment:
Mispelled "bucket" for s3

Reply:
Fixed. a mistery how that happened because test cases are againt correct spelling. Thanks!